OpenID_Connect/oidc_config.sh

```diff
@@ -69,49 +196,31 @@ else
   exit 1
 fi
 
+SETTINGS_ENDPOINT="${URL}/api/admin/auth/settings"
+ACTIVE_ENDPOINT="${URL}/api/auth/settings/active"
+
 if [ "$APP" = "monitor" ] ; then
-  APP_URL="$URL/api/admin/customer/${CUSTOMER_ID}/openid"
+  PRODUCT=""
```

Comment:
where is this variable used?

Reply:
Yeah. I forgot to mention when opening the PR, my very bad. I forgot to mention I had issues while configuring this for secure product. I wanted to ask @popovicgit to know how we have tested this before. Any hint here would be appreciated.

Reply:
Added a comment at `env.sh`